doc/whats_new.rst

```diff
@@ -25,13 +25,15 @@ xxx
 Authors
 ~~~~~~~
 * `Stefan Appelhoff`_
+* `Adam Li`_
 
 Detailed list of changes
 ~~~~~~~~~~~~~~~~~~~~~~~~
 
 Enhancements
 ^^^^^^^^^^^^
 - The function :func:`mne_bids.print_dir_tree` has a new parameter ``return_str`` which allows it to return a str of the dir tree instead of printing it, by `Stefan Appelhoff`_ (`#600 <https://github.com/mne-tools/mne-bids/pull/600>`_)
+- The function :func:`mne_bids.write_raw_bids` now outputs `electrodes.tsv` and `coordsystem.json` files for EEG/iEEG data that are BIDS compliant (only contain subject, session, acquisition, and space entities), by `Adam Li`_ (`#601 <https://github.com/mne-tools/mne-bids/pull/601>`_)
```

Comment:
this probably goes under bug fixes

Reply:
electrodes.tsv should go between double-backticks here